<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MibBrowser</class>
 <widget class="QDialog" name="MibBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIB Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../qHostMonitor.qrc">
    <normaloff>:/img/hostMonitor.png</normaloff>:/img/hostMonitor.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeMibs"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,3,0" columnminimumwidth="0,0,0,0">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOID">
         <property name="text">
          <string>OID</string>
         </property>
         <property name="buddy">
          <cstring>editOid</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editAccess">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblStatus">
         <property name="text">
          <string>STATUS</string>
         </property>
         <property name="buddy">
          <cstring>editStatus</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="editOid"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblMIB">
         <property name="text">
          <string>MIB</string>
         </property>
         <property name="buddy">
          <cstring>editMib</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblAccess">
         <property name="text">
          <string>ACCESS</string>
         </property>
         <property name="buddy">
          <cstring>editAccess</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editSyntax">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editStatus">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="editMib"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSyntax">
         <property name="text">
          <string>SYNTAX</string>
         </property>
         <property name="buddy">
          <cstring>editSyntax</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" rowspan="3" colspan="2">
        <widget class="QPlainTextEdit" name="txtDescription">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBoxMibBrowser">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBoxMibBrowser</sender>
   <signal>accepted()</signal>
   <receiver>MibBrowser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxMibBrowser</sender>
   <signal>rejected()</signal>
   <receiver>MibBrowser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
